mxcache-runtime/src/main/java/com/maxifier/mxcache/hashing/BooleanArrayHashingStrategy.java

```diff
@@ -14,8 +14,6 @@
 
     private static final BooleanArrayHashingStrategy INSTANCE = new BooleanArrayHashingStrategy();

```

Comment:
This logic is in HashingStrategyFactory#createHashingStrategies.
If some hashing strategy is used as a default for some type, it's created by `getInstance()`. If it's declared by `@HashingStrategy`, it's created with a default constructor and put into a static array. So you need an accessible default constructor for all declarable hashing strategies. 

I removed private constructor just in case someone would like to declare it explicitly for some reason (as I do in test).